Library/Formula/notmuch.rb

```diff
@@ -38,8 +39,17 @@ def install
     else
       args << "--without-emacs"
     end
-    system "./configure", *args
 
+    system "./configure", *args
     system "make", "V=1", "install"
+
+    if build.with? "python"
+      Language::Python.each_python(build) do |python, version|
+        Dir.chdir("bindings/python/") do
+          system python, "setup.py", "install"
```

Comment:
This should be `system python, *Language::Python.setup_install_args(prefix)` which adds a wrapper script and some flags around `setup.py` to make our `setup.py` invocations behave more like we want them to.